docs/design/tenancy.md

```diff
@@ -309,6 +309,15 @@ CREATE UNIQUE INDEX idx_deployment_config_project_id_name ON deployment_config(p
 #### Tenant Mode
 This will be a new ENUM field on the existing [project table](https://github.com/bytebase/bytebase/blob/main/store/migration/10001__init_schema.sql#L202).
 
+### Tenancy with different database names
+
+There are use cases that customers put different tenant databases onto the same or a few instances. Since database names are unique within an instance, tenants will have different database names. There are two approaches to solve the problem by reusing most of the infrastructure above.
+
+- Database naming pattern: most users have tenant names as part of database names such as `db1_customer1`, `db1_customer2`, `db2_customer1`, `db2_customer2`. We can allow users to provide a database name pattern e.g. `{{DB_NAME}}_{{TENANT_NAME}}` or `{{DB_NAME}}_{{LOCATION}}` to pin a collection of tenant databases with the same `{{DB_NAME}}` prefix. This name pattern is a project level setting which is allowed only for projects in tenant mode.
```

Comment:
OK, I guess we'd better keep consistency

I personally prefer `{{DB_NAME}}_{{TENANT_NAME}}` and `{{DB_NAME}}_{{LOCATION_NAME}}`

But if you feel it's too verbose, then I am also OK with `{{DB_NAME}}_{{TENANT}}` and  `{{DB_NAME}}_{{LOCATION}}`